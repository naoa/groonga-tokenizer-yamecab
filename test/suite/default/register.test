register tokenizers/yamecab

load --table @yamecab
[
{"_key": "TokenYaMecab", "@updates":true}
]
tokenizer_list
