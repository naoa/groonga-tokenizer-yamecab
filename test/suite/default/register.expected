register tokenizers/yamecab
[[0,0.0,0.0],true]
load --table @yamecab
[
{"_key": "TokenYaMecab", "@updates":true}
]
[[0,0.0,0.0],1]
tokenizer_list
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "name": "TokenMecab"
    },
    {
      "name": "TokenDelimit"
    },
    {
      "name": "TokenUnigram"
    },
    {
      "name": "TokenBigram"
    },
    {
      "name": "TokenTrigram"
    },
    {
      "name": "TokenBigramSplitSymbol"
    },
    {
      "name": "TokenBigramSplitSymbolAlpha"
    },
    {
      "name": "TokenBigramSplitSymbolAlphaDigit"
    },
    {
      "name": "TokenBigramIgnoreBlank"
    },
    {
      "name": "TokenBigramIgnoreBlankSplitSymbol"
    },
    {
      "name": "TokenBigramIgnoreBlankSplitSymbolAlpha"
    },
    {
      "name": "TokenBigramIgnoreBlankSplitSymbolAlphaDigit"
    },
    {
      "name": "TokenDelimitNull"
    },
    {
      "name": "TokenYaMecab"
    }
  ]
]
